### DISCUSSION TITLE
Server doesn't allow dll uploads.

### DISCUSSION THREAD

**Discussion by TheOKey** [Pryaxis/TShock#2239] [Q&A] [answered]
Hey there, we were trying to upload plugins to our server, but the server provider where we host our server doesn't allow .dll file uploads. Is there a way to add plugins in another format?

**Comment by bartico6** [comment, marked as answer]
No, there isn't. Plugins are .dll files and cannot be in any other format.

**Comment by Kojirremer** [comment]
As bartico said, if it's not a DLL, it won't even be registered as a plugin. Do you not have access to the remote desktop? If you have it, it's just a matter of compressing them, upload the archive, then decompress. But I'm guessing that's not the case, right?
